crates/iroha_primitives/src/json.rs

```diff
@@ -3,27 +3,32 @@
 
 #[cfg(not(feature = "std"))]
 use alloc::{
-    format,
+    borrow::ToOwned,
+    collections::BTreeMap,
     string::{String, ToString},
     vec::Vec,
 };
-use core::str::FromStr;
+use core::{cmp::Ordering, str::FromStr};
 #[cfg(feature = "std")]
 use std::{
+    collections::BTreeMap,
     string::{String, ToString},
     vec::Vec,
 };
 
 use derive_more::Display;
-use iroha_schema::IntoSchema;
-use parity_scale_codec::{Decode, Encode};
+use parity_scale_codec::{Compact, Decode, Encode};
 use serde::{de::DeserializeOwned, Deserialize, Serialize};
 use serde_json::Value;
 
 /// A valid `JsonString` that consists of valid String of Json type
-#[derive(Debug, Display, Clone, PartialOrd, PartialEq, Ord, Eq, IntoSchema, Encode, Decode)]
+#[derive(Debug, Display, Clone, Deserialize, Serialize)]
 #[display(fmt = "{_0}")]
-pub struct JsonString(String);
+#[serde(transparent)]
+pub struct JsonString(Value);
```

Comment:
```suggestion
pub struct JsonValue(Value);
```

if we're to proceed with this change we'll have to rename it

Reply:
Yes, made a draft with minimal changes